.circleci/config.yml

```diff
@@ -108,10 +121,10 @@ jobs:
           at: ~/repo/
       - restore_cache:
           keys:
-            - v2-dependencies-{{ checksum "Pipfile.lock" }}
-            - v2-dependencies-default
-      - run: *install-pipenv
-
+            - v3-dependencies-{{ checksum "Pipfile.lock" }}
+            - v3-dependencies-default
+      - run: pip install -U pip pipenv --quiet --no-input
+      - run: pipenv run pipenv shell
```

Comment:
What are we trying to achieve here with `pipenv run pipenv shell` ?
In general we either want to use `pipenv run` to invoke a command in an ad-hoc virtualenv

For example:
```
pipenv run black .
pipenv run isort .
```

or use `pipenv shell` to activate a virtualenv we can run commands in

For example:
```
pipenv shell
black .
isort .
```

`pipenv run pipenv shell` is legal syntax (activate a virtualenv, activate the same virtualenv again, deactivate it), but it doesn't make sense to me as a command to run.

Reply:
I've removed `pipenv run pipenv shell`; I may have added this with my earlier attempt to find a missing venv.